compiler/rustc_target/src/callconv/mod.rs

```diff
@@ -897,6 +898,37 @@ impl FromStr for Conv {
     }
 }
 
+fn conv_to_externabi(conv: &Conv) -> ExternAbi {
+    match conv {
+        Conv::C => ExternAbi::C { unwind: false },
```

Comment:
This loses the distinction between `extern "C"` and `extern "C-unwind"` which could be confusing for users.